src/torch_ucc_init_oss.cpp

```diff
@@ -1,4 +1,19 @@
 #include <torch/python.h>
+#include "torch_ucc.hpp"
 
-PYBIND11_MODULE(TORCH_EXTENSION_NAME, m) {
+// This function is intentionally designed to take a void * argument
```

Comment:
can you just use `export "C"`? it'd disable mangling

Reply:
It works! Thanks! I didn't know this trick.